packages/components/src/components/buttons/IconButton/IconButton.tsx

```diff
@@ -75,10 +78,13 @@ export const IconButton = ({
                 disabled={isDisabled || isLoading}
                 onClick={handleClick}
                 $isSubtle={isSubtle}
+                $borderRadius={rest.borderRadius}
```

Comment:
We should discuss with Ondra, because it doesn't look like typical IconButton component and at the same time we don't want to allow too much of variety for single use cases.

I checked the design and it looks like it's nav item which we have in the sidebar menu. I think we should create one component for these two use cases.

Reply:
Yes, it looks like it's more like nav item, but it was very ineffective to use the related component anyway, lots of differences with styling, colors, etc. I'd have to overwrite lots of stuff, and that's we are trying to get rid of. Found that with `IconButton` it's less code changes, more readable code, even if still not ideal. 

You know, due to Ondra's illness we weren't able to figure out anything else yet.

So do you suggest to create a different component for this use case and not to use the adjusted IconButton, right? Or better to wait and reach out to Ondra for more ideas about the buttons used in this new Welcome screen?